README.md

```diff
@@ -311,19 +311,19 @@ make build_docker
 Run the following command to enter the interactive CLI:
 
 ```sh
-docker run -it --rm -v $(pwd)/validator_keys:/app/validator_keys ethereum/staking-deposit-cli
+docker run -it --rm -v $(pwd)/validator_keys:/app/validator_keys europe-docker.pkg.dev/lks-lz-artifacts/docker-deposit-cli
```

Comment:
Why hosting this on docker pkg? and where is it hosted at ethereum? directly in the repo?